src/SearchBar.js

```diff
@@ -0,0 +1,55 @@
+import React, {Component} from 'react';
+import './SearchBar.css'
+import { Link } from 'react-router-dom';
+
+class SearchBar extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      id: ''
+    }
+  }
+
+  // user selects Title from movie dropdown
+  // hits "Go" button 
+        // need to add button 
+  // go to Movie Display page - new URL
+        
+
+  handleChange = (event) => {
+    // const { id, title } = event.target.value
+    console.log("event", event);
+    // console.log("id", id)
+    this.setState({ 
+      id: event.target.value
+      // title: title
+      // id: event.target.id
+    }
+      
+    )
+  }
+
+  render() {
+    const movieTitles = [...this.props.movies].sort((a,b) => a.title.localeCompare(b.title))
```

Comment:
Using spread operator with array brackets allows the main page to not be alphabetized! Who would have thought?! 😎